apps/vaccitron/src/app/app-header/app-header.component.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Can we split tests and production code to different folders?
(You may ignore this comment if I'm just not used to the conventions of those technologies).

Reply:
This is the convention I work with. There's no difference for me between tests and production code besides the fact that the tests are written before the production code :)